### DISCUSSION TITLE
How to have Xyce installed on Debian stable based Linux

### DISCUSSION THREAD

**Discussion by abdulbadii** [Xyce/Xyce#32] [Q&A] [answered]
How do we have a new or latest Xyce installed on Debian-stable based Linux
just please show common build and install procedure

**Comment by tvrusso** [comment, marked as answer]
In order to install Xyce on a Debian based system you will have to build it yourself according to the building guide at https://xyce.sandia.gov/documentation/BuildingGuide.html, which documents the common build and install procedures.
